File: src/my_first_node.cpp
#include <ros/ros.h>


int main( int argc, char **argv){
  
  	ros::init(argc, argv, "my_first_cpp_node");    // Initialize the ROS node with the name 'my_first_cpp_node'
  
  	ros::NodeHandle nh;                             // Create a node handle to interact with ROS
  
  	ROS_INFO("Node has been started");              // Log an informational message
  
  	ros::Rate rate(10);
  
	  while( ros::ok()){       // Run the loop until the node is shut down
		  ROS_INFO("Hello");     // Log an informational message
		  rate.sleep();          // Sleep for the remainder of the 0.1 second loop iteration
	  }
  
  }

File: catkin_ws/src/my_robot_tutorials/scripts/my_first_node.py
#!/usr/bin/env python3

import rospy


if __name__ == '__main__':

  # Initialize a ROS node with the name 'my_first_python_node'
  rospy.init_node('my_first_python_node')

   # Log an informational message
   rospy.loginfo("This node has been started")


  rate = rospy.Rate(10)

 
  while not rospy.is_shutdown():       # Run the loop until the node is shutdown
    	rospy.loginfo("Hello")           # Log an informational message
    	rate.sleep()                     # Sleep for the remainder of the 0.1 second loop iteration
